### DISCUSSION TITLE
Backstep

### DISCUSSION THREAD

**Discussion by djstrobe13** [bcholmes/StarTrek2d20#167] [Ideas]
May I suggest adding into the Star trek Adventures creator system the ability to return to previous steps. At the moment, if you change your mind about a step in the character creation you have to start over from the very beginning.

**Comment by bcholmes** [comment]
Two points that are worth mentioning:

1. You don't have to start off at the very beginning. You can use the "history" item in the side bar to jump back to a previous spot in the character creation process. It's still true that you have to re-do everything from that point onward, though.
2. The way I implemented Solo characters is a mechanism that I plan to roll out to the main character creation: in that case, you can use the breadcrumbs (or history) to jump back to a previous step and change a past decision, and still keep your later choices intact (with some restrictions).

A related thing that would be ideal: enabling the back button. It's on my list of things to look at.

**Reply by QosmicQi** [reply]
Not sure if this is just user error, but when I use the History menu to revisit a step, I experience a ton of glitches: the attribute points don't seem to reset, so it looks like I can dump more points than technically allowed; sometimes trying to redo career events does not work, and it opens up the previously chosen event in stead of the new one; and when I get to the end, the entire thing often crashes into a black screen. I usually have to start over anyway. 

(That said, this is an excellent tool, and I love how flexible it is)

**Comment by djstrobe13** [comment]
Brilliant! thanks for the feedback!
    On Monday, 11 September 2023 at 22:26:27 BST, BC Holmes ***@***.***> wrote:  
 
 


Two points that are worth mentioning:
   
   - You don't have to start off at the very beginning. You can use the "history" item in the side bar to jump back to a previous spot in the character creation process. It's still true that you have to re-do everything from that point onward, though.
   - The way I implemented Solo characters is a mechanism that I plan to roll out to the main character creation: in that case, you can use the breadcrumbs (or history) to jump back to a previous step and change a past decision, and still keep your later choices intact (with some restrictions).

A related thing that would be ideal: enabling the back button. It's on my list of things to look at.
